test/dump/except.txt

```diff
@@ -0,0 +1,37 @@
+;;; TOOL: run-objdump
+;;; ARGS0: -v
+;;; ARGS1: -x
+(module
+  (except)
+  (except i32)
+  (except f32 f64))
+(;; STDOUT ;;;
+0000000: 0061 736d                                 ; WASM_BINARY_MAGIC
+0000004: 0100 0000                                 ; WASM_BINARY_VERSION
+; section "exception"
+0000008: 00                                        ; section code
+0000009: 00                                        ; section size (guess)
+000000a: 09                                        ; string length
+000000b: 6578 6365 7074 696f 6e                   exception  ; custom section name
+0000014: 03                                        ; exception count
+0000015: 00                                        ; exception type count
+0000016: 01                                        ; exception type count
+0000017: 7f                                        ; i32
+0000018: 02                                        ; exception type count
+0000019: 7d                                        ; f32
+000001a: 7c                                        ; f64
+0000009: 11                                        ; FIXUP section size
+
+except.wasm:	file format wasm 0x1
+
+Section Details:
+
+Custom:
```

Comment:
Does this mean the exception section is being treated as a custom section?

Reply:
It looks like that's what the proposal currently has: https://github.com/WebAssembly/exception-handling/blob/master/proposals/Level-1.md#exception-section

Reply:
Ah, okay. I think the exception section should be a Wasm section like say the types sections. I thought the idea behind custom sections was that they could be ignored or dropped without changing the semantics of the module, which isn't the case for exceptions.

We should probably clarify this in the exceptions proposal.

Reply:
Yeah, there was discussion about this previously. Can't remember where, though. I seem to recall an issue being that this would be the first section number that is out of order. But I don't think that's a big deal.

Reply:
It likes like the discussion was at https://github.com/WebAssembly/exception-handling/issues/25 and https://github.com/WebAssembly/exception-handling/pull/27.

I think the idea was that future sections would have out of order IDs, but that's by design. Having a fixed order was more important than having them in numerical order.